### PR TITLE
Feature/6 control platform

### PR DESCRIPTION
**Pull request by lisset-orozco**
- [x] config google_oauth2

- [x] save sessions

- [x] get logged in user

- [x] close session

- [x] scaffold and validations of the model for applications

- [x] fixing the views

- [x] fixing the tokens

- [x] spec and controller my_apps

- [x] sessions in the tests

- [x] fixing controller

- [x] show apps

- [x] validate secret_key

- [x] validate origins_ajax

- [x] tokens by application and creation of the my_app service

### CODE REVIEW

File: app/controllers/my_apps_controller.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  before_action :set_my_app, only: %i[edit update destroy]
+  before_action :authenticate_user!
+
+  # GET /my_apps/new
+  def new
+    @my_app = MyApp.new
+  end
+
+  # GET /my_apps/1/edit
+  def edit
```
Comment (joalbertg) [inline app/controllers/my_apps_controller.rb]: only line please

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# welcome controller
+class WelcomeController < ApplicationController
+  def index
```
Comment (joalbertg) [inline app/controllers/welcome_controller.rb]: only line please

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -2,4 +2,13 @@
 
 # application helper
 module ApplicationHelper
+  def user_signed_in?
+    # true if the user is logged in otherwise false
+    !current_user.nil?
```
Comment (joalbertg) [inline app/helpers/application_helper.rb]: and method `blank?`

**Review by joalbertg — CHANGES_REQUESTED** [review]

File: app/controllers/my_apps_controller.rb (hunk 1 above)
Comment (lisset-orozco) [reply app/controllers/my_apps_controller.rb]: Done

File: app/controllers/welcome_controller.rb (hunk 2 above)
Comment (lisset-orozco) [reply app/controllers/welcome_controller.rb]: Done

File: app/helpers/application_helper.rb (hunk 3 above)
Comment (lisset-orozco) [reply app/helpers/application_helper.rb]: done

**Review by joalbertg — APPROVED** [review]

**Pull request merged**